src/ansys/pyensight/core/utils/parts.py

```diff
@@ -26,12 +26,10 @@
     from ansys.pyensight.core.listobj import ensobjlist
 
 if TYPE_CHECKING:
-    try:
-        from ensight.objs import ENS_PART, ENS_VAR  # type: ignore
-    except ImportError:
-        from ansys.api.pyensight import ensight_api
-        from ansys.api.pyensight.ens_part import ENS_PART
-        from ansys.api.pyensight.ens_var import ENS_VAR
+    from ansys.api.pyensight import ensight_api
```

Comment:
This is nice. I guessed why I was adding in the type checking area the pure ensight classes. VSCode has no idea of what they are, and so using "ENS_PART" or "ENS_VAR" as type hint values was making no effect.
Instead, since types are really needed only in PyEnSight, I can easily use directly the PyEnSight definitions for types. Indeed these are available even in EnSight, so if one sets its default interpreter in VSCode to be the EnSight Python interpreter, types would come out anyway